engine/env/spark/src/main/java/org/datacleaner/spark/SparkJobContext.java

```diff
@@ -59,6 +62,8 @@
     private static String DATA_CLEANER_RESULT_PATH_PROPERTY = "datacleaner.result.hdfs.path";
     public static final String ACCUMULATOR_CONFIGURATION_READS = "DataCleanerConfiguration reads";
     public static final String ACCUMULATOR_JOB_READS = "AnalysisJob reads";
+    private static String DEFAULT_RESULT_PATH = "/datacleaner/results/";
+    private static String RESULT_FILE_EXTENSION = ".analysis.result.dat";
```

Comment:
Make sure to also add "final" to static properties. I see this also needed a few lines above (although not part of this PR).